### DISCUSSION TITLE
Independent upgrade of the webui

### DISCUSSION THREAD

**Discussion by zhubao315** [AbdBarho/stable-diffusion-webui-docker#499] [Ideas]
"The webui has been upgraded to version 1.3.1, do you consider upgrading the docker version separately?"

https://github.com/AUTOMATIC1111/stable-diffusion-webui/releases/tag/v1.3.1

**Comment by AbdBarho** [comment]
I will update it soon!

**Reply by AbdBarho** [reply]
There are bugs with control net so I will wait a bit

https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/baf6946e06249c5af9851c60171692c44ef633e0
